tests/regression.rs

```diff
@@ -237,12 +238,12 @@ fn test_affine_nearest_rgb() {
     fn affine_nearest(image: &RgbImage) -> RgbImage {
         let root_two_inv = 1f32 / 2f32.sqrt() * 2.0;
         #[rustfmt::skip]
-        let hom = Projection::from_matrix([
-            root_two_inv, -root_two_inv,  50.0,
-            root_two_inv,  root_two_inv, -70.0,
-                     0.0,           0.0,   1.0,
+            let hom = Projection::from_matrix([
```

Comment:
This indentation looks wrong

Reply:
The indentation of these three functions still looks wrong - “let hom” is indented further than the rustfmt attribute above it.